packages/mercury/docs/installation-for-angular.md

```diff
@@ -0,0 +1,344 @@
+
+</details>
+
+### 3.4. Style the components with the bundles
+
+`custom-dialog.ts` <small>(Example component)</small>
+
+```ts
+import {
+  ChangeDetectionStrategy,
+  Component,
+  CUSTOM_ELEMENTS_SCHEMA
+} from "@angular/core";
+
+import { getBundles } from "@genexus/mercury/bundles.js";
+import { getIconPath } from "@genexus/mercury/assets-manager.js";
+
+@Component({
+  selector: "custom-dialog",
+  standalone: true,
+  styleUrl: "./custom-dialog.scss",
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
+  template: `<ch-theme [model]="bundles"></ch-theme>
+    <button class="button-primary button-icon-and-text" type="button">
+      <ch-image class="icon-md" [src]="ICON" [type]="'mask'" />
+      Caption
+    </button>`
+})
+export class CustomDialogComponent {
+  bundles = getBundles(
+    ["components/button", "components/icon"],
+    "{{ CSS bundles final path }}"
+  );
+  ICON = getIconPath({
+    category: "system",
+    name: "add-circle",
+    colorType: "primary"
+  });
+}
+```
+
+<details>
+<summary>Code explained:</summary>
+<br>
+
+The preceding code is from a dummy component called "custom-dialog". It includes a button element with an icon. By importing the `getBundles` utility, we are able to get the button and the icon bundles `["components/button", "components/icon"]`. Then, by importing the `getIconPath` utlity, we are able to get the icon path/source.
+
+The `ch-theme`, is the [Chameleon component](https://github.com/genexuslabs/chameleon-controls-library/tree/main/src/components/theme), a component for downloading and using the CSS bundles in the application.
+
+</details>
+
+## 4. Copy bundles and assets
+
+Copy the CSS bundles, custom fonts and icons to the dev and prod builds. Custom fonts and icons are distributed under the following folders:
+
+- `<path to node_modules>/@genexus/mercury/dist/assets/fonts`
+- `<path to node_modules>/@genexus/mercury/dist/assets/icons`
+
+To use these assets with the CSS bundles, we recommend copying them statically, that is, do not track these assets in your repository sources, only copy them to the dev server and prod builds:
+
+`angular.json`
+
+```json
+{
+  "projects": {
+    "<app name>": {
+      "architect": {
+        "build": {
+          "options": {
+            "assets": [
+              {
+                "glob": "**/*",
+                "input": "{{ CSS outDir path }}",
+                "output": "{{ CSS bundles final path }}"
+              },
+              {
+                "glob": "**/*",
+                "input": "<path to node_modules>/@genexus/mercury/dist/assets/fonts",
+                "output": "{{ Fonts final path }}"
+              },
+              {
+                "glob": "**/*",
+                "input": "<path to node_modules>/@genexus/mercury/dist/assets/icons",
+                "output": "{{ Icons final path }}"
+              }
+            ]
+          }
+        }
+      }
+    }
+  }
+}
+```
+
```

Comment:
This section should be placed right after the CSS bundle creation, where we explain the paths and run the build command to create the CSS files. This was the case in the original (and not so clear) readme file.